docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -28,16 +28,16 @@ The Miden client facilitates interaction with the Miden rollup and provides a wa
 3. Initialize the client. This creates the `miden-client.toml` file.
 
 ```shell
-miden init --rpc xxx.xxx.xxx.xxx
+miden init --rpc 18.203.155.106
```

Comment:
@Dominik1999, @phklive - is there something we need to do on our end to enable using something like `testnet.miden.io` here?

Reply:
last time I checked, DNS resolution was resulting in a different IP (which I checked and didn't have the corresponding ports open). Should that be fixed I think we can replace it without changes on the client